en/assembly/quick_start_durandal.md

```diff
@@ -0,0 +1,178 @@
+  
+  ![Durandal - DSM](../../assets/flight_controller/durandal/dsm.jpg)
+
+- PPM and S.Bus receivers connect to the [SBUS_IN/PPM_IN](../flight_controller/durandal.md#rc-in) input port (marked as RC IN, next to the MAIN/AUX inputs).
+  
+  ![Durandal - Back Pinouts (Schematic)](../../assets/flight_controller/durandal/durandal_pinouts_back.jpg)
+
+- PPM and PWM receivers that have an *individual wire for each channel* must connect to the **PPM RC** port *via a PPM encoder* [like this one](http://www.getfpv.com/radios/radio-accessories/holybro-ppm-encoder-module.html) (PPM-Sum receivers use a single signal wire for all channels).
+
+For more information about selecting a radio system, receiver compatibility, and binding your transmitter/receiver pair, see: [Remote Control Transmitters & Receivers](../getting_started/rc_transmitter_receiver.md).
+
+
+## Telemetry Radios (Optional)
+
+Telemetry radios may be used to communicate and control a vehicle in flight from a ground station (for example, you can direct the UAV to a particular position, or upload a new mission).
+
+The vehicle-based radio should be connected to the [TELEM1](../flight_controller/durandal.md#telem1_2_3) port as shown below using one of the 6-pos connectors (if connected to this port, no further configuration is required).
+The other radio is connected to your ground station computer or mobile device (usually by USB).
+
+![Durandal/Telemetry Radio](../../assets/flight_controller/durandal/holybro_telemetry_radio.jpg)
+
+
+## SD Card (Optional)
+
+SD cards are most commonly used to [log and analyse flight details](../getting_started/flight_reporting.md).
+Insert a card into the *Durandal* where indicated below.
+
+![Durandal SD Card](../../assets/flight_controller/durandal/durandal_sd_slot.jpg)
+
+> **Tip** The SanDisk Extreme U3 32GB is [highly recommended](https://dev.px4.io/master/en/log/logging.html#sd-cards) (Developer Guide).
+
+
+## Motors
+
+Motors/servos are connected to the **I/O PWM OUT** (**MAIN OUT**) and **FMU PWM OUT** (**AUX**) ports in the order specified for your vehicle in the [Airframe Reference](../airframes/airframe_reference.md). 
+
+![Durandal - Back Pinouts (Schematic)](../../assets/flight_controller/durandal/durandal_pinouts_back.jpg)
+
+
+> **Note** This reference lists the output port to motor/servo mapping for all supported air and ground frames (if your frame is not listed in the reference then use a "generic" airframe of the correct type). 
+
+<span></span>
+> **Note** There are 5 AUX ports; you cannot use this flight controller for airframes that use AUX6, AUX7, AUX8 for motors or other critical flight controls.
+
+
+## Other Peripherals
+
+The wiring and configuration of optional/less common components is covered within the topics for individual [peripherals](../peripherals/README.md).
+
+## Pinouts
+
+[Durandal > Pinouts](../flight_controller/durandal.md#pinouts)
+
+## Configuration
```

Comment:
@bkueng The FC comes with APM onboard. Do we perhaps spell out first step is to update firmware to PX4?

Reply:
Yes we can do that.